app/src/main/java/nyc/c4q/jonathancolon/inContaq/utlities/dailysms/DailySentWorkerTask.java

```diff
@@ -0,0 +1,76 @@
+package nyc.c4q.jonathancolon.inContaq.utlities.dailysms;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.joda.time.DateTime;
+
+import java.util.ArrayList;
+import java.util.TreeMap;
+
+import nyc.c4q.jonathancolon.inContaq.utlities.sms.Sms;
+
+/**
+ * Created by Hyun on 3/11/17.
+ */
+
+public class DailySentWorkerTask extends AsyncTask<DailyTaskParams, Void,TreeMap<Integer, Integer>> {
```

Comment:
From the name I expected this to be pretty much the same as DailyReceivedWorkerTask, but it's not... why?